### ISSUE TITLE
Segmentation fault on graph-cli usage

### ISSUE THREAD

**Issue by nisarg-soni** [graphprotocol/graph-cli#980]
First of all on trying installation with `npm`, with the command : 
`sudo npm install -g @graphprotocol/graph-cli`
It throws an error :
![Screenshot from 2022-10-21 12-49-06](https://user-images.githubusercontent.com/32036765/197137101-e7a853d5-889b-42ea-be35-067cc5338892.png)

So I decided to try installing with `yarn` and it worked with the output :
![Screenshot from 2022-10-21 12-49-59](https://user-images.githubusercontent.com/32036765/197137233-72b9f708-7786-4235-ab61-33fe21006383.png)

but then when trying to use it, like running `graph init` it throws a segmentation fault :
![Screenshot from 2022-10-21 12-50-22](https://user-images.githubusercontent.com/32036765/197137752-39b6d971-42e3-42e4-baca-699d6e4c54a5.png)

`yarn version : 1.22.19`
`npm version : 8.19.2`
`node version : v16.18.0`

`OS : Ubuntu 22.04.1 LTS`

Could someone please help with this?

**Comment by azf20** [comment]
hi! have you tried using a local installation in the repository you are working on? (i.e. does this only happen with the global install)

**Comment by nisarg-soni** [comment]
Yeah it works on a local installation . Only happens with the global installation
